src/core/Akka.Persistence/Serialization/PersistenceMessageSerializer.cs

```diff
@@ -203,18 +202,15 @@ public override string Manifest(object obj)
             switch (obj)
             {
                 case IPersistentRepresentation _:
-                    return IPersistentRepresentationManifest;
                 case AtomicWrite _:
-                    return AtomicWriteManifest;
                 case AtLeastOnceDeliverySnapshot _:
-                    return AtLeastOnceDeliverySnapshotManifest;
                 case PersistentFSM.StateChangeEvent _:
-                    return StateChangeEventManifest;
+                    return obj.GetType().TypeQualifiedName();
```

Comment:
Would something like this be OK?

Reply:
I'd have problem with `Akka.Persistence.Fsm.PersistentFSM.PersistentFSMSnapshot` because it is a generic class, hard coding would not work